src/openforms/registrations/contrib/zgw_apis/plugin.py

```diff
@@ -401,6 +438,31 @@ def register_submission(self, submission: Submission, options: dict) -> dict | N
         if submission.has_registrator:
             result["medewerker_rol"] = medewerker_rol
 
+        # Register submission to Objects API if configured
+        if (
+            options.get("objecttype")
+            and options.get("objecttype_version")
+            and options.get("content_json")
+        ):
```

Comment:
```suggestion
        if (
            (object_type := options.get("objecttype"))
            and (object_type_version := options.get("objecttype_version"))
            and (content_json := options.get("content_json"))
        ):
```
and now you can use those new variables inside the `if`-block, so you no longer need to repeat the `options.get(...)` calls :)